packages/config/src/layer2s/zksynclite.ts

```diff
@@ -167,18 +174,20 @@ export const zksynclite: Layer2 = {
   },
   contracts: {
     addresses: [
-      discovery.getMainContractDetails(
-        'ZkSync',
-        `The main Rollup contract. Allows the operator to commit blocks, provide zkProofs (validated by the Verifier) and processes withdrawals by executing blocks. Users can deposit ETH and ERC20 tokens. This contract also defines the upgrade process for all the other contracts by enforcing an upgrade delay and employing the Security Council which can shorten upgrade times.`,
-      ),
-      discovery.getMainContractDetails(
-        'Verifier',
-        'Implements zkProof verification logic.',
-      ),
-      discovery.getMainContractDetails(
-        'Governance',
-        'Keeps a list of block producers, NFT factories and whitelisted tokens.',
-      ),
+      discovery.getMainContractDetails('ZkSync', {
+        description:
+          'The main Rollup contract. Allows the operator to commit blocks, provide zkProofs (validated by the Verifier) and processes withdrawals by executing blocks. Users can deposit ETH and ERC20 tokens. This contract also defines the upgrade process for all the other contracts by enforcing an upgrade delay and employing the Security Council which can shorten upgrade times.',
+        ...upgrades,
+      }),
+      discovery.getMainContractDetails('Verifier', {
+        description: 'Implements zkProof verification logic.',
+        ...upgrades,
+      }),
+      discovery.getMainContractDetails('Governance', {
+        description:
+          'Keeps a list of block producers, NFT factories and whitelisted tokens.',
+        ...upgrades,
```

Comment:
I'm just thinking if this information should be here or in the permission addresses section... That's a lot of repeated info.